enclave/src/utils.rs

```diff
@@ -117,13 +156,32 @@ pub fn read_counterstate(filepath: &str) -> SgxResult<Vec<u8>> {
 			}
 		},
 		Err(x) => {
-			error!("[Enclave] can't get counter file! {}", x);
+			error!("[Enclave] can't get counter file, initializing new Counter! {}", x);
```

Comment:
please separate concerns:
If you call this function `read_plaintext()` it shouldn't do anything else. It shouldn't know that it reads a counter value.